xml/depl_maintenance.xml

```diff
@@ -1503,6 +1503,304 @@ migrate</screen>
+       <para>
+	Node has failed during OS upgrade.
+       </para>
+      </question>
+      <answer>
+       <para>
+	Possible reasons include an incorrect repository setup or package
+	conflicts. Check the <filename>/var/log/crowbar/node-upgrade.log</filename> log file on the
+	affected node. Check the repositories on node using the <command>zypper
+	lr</command> command. Make sure the required repositories are
+	available. To test the setup, install a package manually or run the
+	<command>zypper dup</command> command (this command is executed by the
+	upgrade script). Fix the repository setup and run the failed upgrade
+	step again. If custom package versions or version locks are in place,
+	make sure that they don't interfere with the <command>zypper dup</command> command.
+       </para>
+      </answer>
+     </qandaentry>
+     <qandaentry>
+      <question>
+       <para>
+	Node does not come up after reboot.
+       </para>
+      </question>
+      <answer>
+       <para>
+	It could happen that it takes the node too long to come up, which cause
+	a timeout. Check the node manually, make sure it is online, and repeat the step.
+       </para>
+      </answer>
+     </qandaentry>
+     <qandaentry>
+      <question>
+       <para>
+	N number of nodes were provided to compute upgrade using
+	<command>crowbarctl upgrade nodes node_1,node_2,...,node_N</command>,
+	but less then N were actually upgraded.
+       </para>
+      </question>
+      <answer>
+       <para>
+	If the live migration cannot be performed for certain nodes due to a timeout,
+	&crow; upgrades only the nodes that it was able to
+	live-evacuate in the specified time. Because some nodes have been upgraded, it is possible that
+	more resources will be available for live-migration when you try to run this
+	step again. See also question on live migration failure.
+       </para>
+      </answer>
+     </qandaentry>
+     <qandaentry>
+      <question>
+       <para>
+	Node has failed at the initial chef client run stage.
+       </para>
+      </question>
+      <answer>
+       <para>
+	An unsupported entry in the configuration file may prevent a service
+	from starting. This, in turn, causes the node to fail at the initial
+	chef client run stage. Checking the
+	<filename>/var/log/crowbar/crowbar_join/chef.*</filename> log files on
+	the node is a good starting point.
+       </para>
+      </answer>
+     </qandaentry>
+     <qandaentry>
+      <question>
+       <para>
+	I need to change &ostack-bare; configuration during the upgrade but I cannot access &crow;.
+       </para>
+      </question>
+      <answer>
+       <para>
+	&crow; &wi; is accessible only when upgrade is completed or
+	when it is postponed. Postponing the upgrade can be done only after
+	upgrading all &contrnode;s using the <command>crowbarctl upgrade nodes
+	postpone</command> command. You can then access &crow; and
+	even save some modification. It is then necessary to resume the upgrade
```

Comment:
"You can then access &crow; and save your modifications. Before you can continue with the upgrade of rest of the nodes, resume the upgrade using the ...."